### DISCUSSION TITLE
How is the navigation overflow supposed to work?

### DISCUSSION THREAD

**Discussion by rhwood** [mmistakes/minimal-mistakes#2957] [Q&A] [answered]
Is the navigation overflow supposed to turn into an X when open and remain a hamburger (three bars) when closed? Checking out master of this repo, it resolutely remains a hamburger when clicked, and I can't tell if its my browsers (Safari, Firefox, and Microsoft Edge) or if the code to perform that is just left over and not supposed to function.

**Comment by mmistakes** [comment, marked as answer]
Looks broken to me. The three bars are suppose to rotate and turn into an X. The [CSS is there](https://github.com/mmistakes/minimal-mistakes/blob/641ca6f3d8cd8fb0ae24e4f77b2e62a44a65e53b/_sass/minimal-mistakes/_utilities.scss#L363) for it but my quick look at demo site shows it's not setting `.close` on `.navicon` like it used.

I'd have to look back at some commits, but there was a pull request some time ago that made changes to the greedy navigation. I wouldn't be surprised if that broke something then and was just missed.

**Reply by mmistakes** [reply]
Yup, definitely was missing a line of JavaScript that toggled the class when the menu button is clicked. Surprised it took 11 months for someone to notice it was broken!

Fixed it in https://github.com/mmistakes/minimal-mistakes/commit/ee67ad97431a71e334ff7f4ff33ebce59bffedd2
